server/manifestmergetool/src/test/java/com/defold/manifestmergetool/ManifestMergeToolTest.java

```diff
@@ -150,12 +138,9 @@ private String readFile(File file) throws IOException {
     }
 
     @Test
+    @Tag("android")
     public void testMergeAndroid() throws IOException {
-        if (platform != Platform.ANDROID) {
```

Comment:
No need to use `Parametrized` test here and below because every test has precondition for run only for one platform.